api/server.js

```diff
@@ -33,7 +33,8 @@ const db = mysql.createPool({
 	port: process.env.DB_PORT || '8889',
 	user: process.env.DB_USERNAME || 'admin',
 	password: process.env.DB_PASSWORD || 'root',
-	database: process.env.DB_NAME || DB_NAME
+	database: process.env.DB_NAME || DB_NAME,
+	multipleStatements: true
```

Comment:
what does multipleStatements do?

Reply:
Allows for multiple SQL statements to be run in the same db.query() call.